aries-site/src/layouts/navigation/HighlightPhrase.js

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Text, ThemeContext } from 'grommet';
+
+export const HighlightPhrase = ({ children, phrase, ...rest }) => {
+  const theme = useContext(ThemeContext);
+  const regexp = new RegExp(phrase, 'ig');
+  const matches = [...children.matchAll(regexp)];
+  const nextPhrase = [];
+  let nextIndex = 0;
+
+  if (matches.length) {
+    matches.forEach((match, index) => {
+      nextPhrase.push(match.input.slice(nextIndex, match.index));
+      nextIndex = match.index + match[0].length;
+      nextPhrase.push(
+        <Text
+          key={index}
+          weight="bolder"
+          color={
+            theme.dark
+              ? theme.global.colors.yellow.light
+              : theme.global.colors.blue.dark
+          }
```

Comment:
why are we using `yellow` for dark then `blue` for light?

Reply:
Colors are selected for ability to effectively highlight with enough contrast for legibility. 

Yellow on black has strong contrast. Yellow on white, not so much, so need another another color to use. I could not find a color variant which performed well in both contexts, so used two colors.

Open to alternative suggestions.

Reply:
Let me look at the colors and see Ill suggest if I find something else

Reply:
I looked through the colors that we offer and had tried `orange` and `teal` and your correct the contrast was not as good so I think we are okay with keeping as is with the `blue` and `yellow` I was just looking for a consistent color but did not find a great one for both